src/api_calls/orders_calls.js

```diff
@@ -76,3 +76,28 @@ export function DeleteOrder(id){
 
 }
 
+export function UpdateOrder(id,patchRequestBody){
+    console.log('inainte de fetch post'+JSON.stringify(patchRequestBody));
```

Comment:
Please remove this `console.log`.

Reply:
Intrebare, sterg toate console.log-urile din tot proiectul?:)) Si alea din catch(error ...)